deployment/helm/Chart.yaml

```diff
@@ -44,8 +44,16 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.0-beta
+version: 0.4.0-beta
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
 appVersion: latest
+
+dependencies:
+  - name: postgresql
```

Comment:
This is perhaps best suited to a separate PR but I'll mention it here and you can decide:

Helm stable Repo is getting [deprecated](https://github.com/helm/charts#status-of-the-project) in favour of a distributed model. I think both these charts are maintained by Bitnami so we should switch over to [their repository](https://github.com/bitnami/charts) directly at some point.

Reply:
Created a separate issue for this one: https://github.com/pivotal/postfacto/issues/248